src/context/sortable-context.tsx

```diff
@@ -0,0 +1,51 @@
+import { createContext, useContext, useMemo, ReactNode } from 'react'
+import { DraggableAttributes, DraggableSyntheticListeners } from '@dnd-kit/core'
+import { useSortable } from '@dnd-kit/sortable'
+
+interface SortableItemProviderProps {
+  children: ReactNode
+  id: string
+}
+
+interface SortableItemProvideContext {
+  attributes: DraggableAttributes
+  listeners: DraggableSyntheticListeners
+  ref: (node: HTMLElement | null) => void
+}
+
+const SortableItemContext = createContext<SortableItemProvideContext>({
+  attributes: {
+    role: 'button',
+    tabIndex: 0,
+    'aria-disabled': false,
+    'aria-pressed': false,
+    'aria-roledescription': '',
+    'aria-describedby': ''
```

Comment:
Can you please explain what do these attributes do?

Reply:
It's required for  interface `DraggableAttributes` and without it drag&drop will not work